aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -224,6 +224,13 @@ Services that interact with requests from the Internet or a corporate network an
 
 Specify a [Kestrel server HTTPS endpoint configuration](xref:fundamentals/servers/kestrel#endpoint-configuration).
 
+## Current directory and content root
+
+The current working directory returned by calling `Directory.GetCurrentDirectory()` for a Windows Service is the `C:\WINDOWS\system32` folder. The *system32* folder isn't a suitable location to store a service's files (for example, settings files). Use one of the following approaches to maintain and access a service's assets and settings files with [FileConfigurationExtensions.SetBasePath](/dotnet/api/microsoft.extensions.configuration.fileconfigurationextensions.setbasepath) when using an [IConfigurationBuilder](/dotnet/api/microsoft.extensions.configuration.iconfigurationbuilder):
```

Comment:
I've been italicizing folder paths. Consider doing that for `C:\WINDOWS\system32`.